### PR TITLE
Update/job page

### PR DESCRIPTION
**Pull request by cosden**
This PR addresses issues #28 and #29 via:

- A google form to take in the (as of now) required information to post a job to the board
- A policy page that spells out requirements/expectations for jobs posted to the job board

Both are linked from the jobs page.

As part of the review, if someone would give the google form a test run. I've set up email notification for each time the form is completed.

### CODE REVIEW

File: pages/opportunities/job-policy.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: US-RSE Job Posting Policy
+permalink: /job-policy/
```
Comment (vsoch) [inline pages/opportunities/job-policy.md:4]: This should be under the family of jobs, e.g., /jobs/policy/ https://vsoch.github.io/usrse.github.io/jobs/ which means you'll need to update the menu metadata in _data.

File: pages/opportunities/job-policy.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+## Posting RSE-related Jobs on the US-RSE Job Board
```
Comment (vsoch) [inline pages/opportunities/job-policy.md:7]: For this template you will just want to use the title in the front end matter - this one would be redundant.

**Comment by mtbc** [comment]
Not quite this PR really but for the Google form I'd suggest some reformulation of:

> Jobs are currently displayed until the expiration date, hyperlinked to the job URL, with the following information

When I first read it, I was trying to parse it that the following information was going to be used to hyperlink the expiration date to the job URL and I wondered what on Earth. Took me a minute to realize that this is a list of separate things which I *think* are three alternate suffixes to the "displayed"?

File: pages/opportunities/job-policy.md (hunk 1 above)
Comment (cosden) [reply pages/opportunities/job-policy.md:4]: If I understand correctly what you mean, I had debated this myself. I ended up deciding to only link to it from the bottom of the job page. My feeling was that so few people will actually care or read it, that putting it in the menu will only detract from the main attraction: the jobs page. 
Thoughts? (Or did I misinterpret?)

File: pages/opportunities/job-policy.md (hunk 2 above)
Comment (cosden) [reply pages/opportunities/job-policy.md]: Makes sense. Made the change!

**Comment by cosden** [comment]
Thanks @mtbc. Lol. I can see why it didn't make sense. Is this any clearer?

> Jobs are currently displayed until the URL is unreachable or the provided expiration date is reached.
> Job postings include a hyperlink to the supplied job URL along with the Job Title, Organization, and Location.

**Comment by mtbc** [comment]
It certainly is, thank you. :smile: :+1:

File: pages/opportunities/job-policy.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: US-RSE Job Posting Policy
+permalink: /jobs/job-policy/
```
Comment (vsoch) [inline pages/opportunities/job-policy.md:4]:
```suggestion
permalink: /jobs/policy/
```

**Review by vsoch — APPROVED** [review]

File: pages/opportunities/jobs.md [hunk 4]
```diff
@@ -17,4 +17,4 @@ permalink: /jobs/
 <br>
 
 ### Have an RSE-related job posting?  
-Contact us via [slack](https://usrse.slack.com) or directly make a pull request in the website's [GitHub repository](https://github.com/USRSE/usrse.github.io).
+Please read our [job posting policy]({{ site.baseurl }}/job-policy/) first, then fill out this [google form](https://docs.google.com/forms/d/e/1FAIpQLSfYK64R1c0rj-ERldGLxuqedLIbsYPZXj9uBplDRYNmnND10Q/viewform?usp=sf_link) to request additions to the job board.
```
Comment (vsoch) [inline pages/opportunities/jobs.md:20]:
```suggestion
Please read our [job posting policy]({{ site.baseurl }}/jobs/policy/) first, then fill out this [google form](https://docs.google.com/forms/d/e/1FAIpQLSfYK64R1c0rj-ERldGLxuqedLIbsYPZXj9uBplDRYNmnND10Q/viewform?usp=sf_link) to request additions to the job board.
```

Comment (cosden) [reply pages/opportunities/jobs.md:20]: Good catch. Thanks @vsoch!

**Pull request merged**